cross/minio/Makefile

```diff
@@ -1,22 +1,22 @@
 PKG_NAME = minio
-PKG_VERS = 2020-11-25T22-36-25Z
+PKG_VERS = 2021-09-18T18-09-59Z
```

Comment:
i suppose you wanted to define `PKG_VERS = 2021-09-18T18:09:59Z` or the following subst is obsolete...

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/4683/commits/cdf18f1267705853e6b7608837ba0e7dd5bcb568